src/native/_auth.js

```diff
@@ -43,7 +43,7 @@ const ContainerPermissionsArray = new ArrayType(ContainerPermissions);
 
 /// For use in `ShareMDataReq`. Represents a specific `MutableData` that is being shared.
 const ShareMData = Struct({
-  type_tag: t.u64,
+  typeTag: t.u64,
```

Comment:
I'm not sure if it really matters but perhaps is worthy keeping the prop name here as type_tag as it's supposed to match the native struct.